### PR TITLE
Split up

### PR DESCRIPTION
**Pull request by jrleeman**
Initial stab at splitting things up into individual modules. Probably need to discuss naming some. I think this is about the right level of granularity?

### CODE REVIEW

File: examples/defining_new_state_relation.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import numpy as np
 import matplotlib.pyplot as plt
```
Comment (dopplershift) [inline examples/defining_new_state_relation.py]: no longer need `plt`

File: examples/slide_hold_slide.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import matplotlib.pyplot as plt
```
Comment (dopplershift) [inline examples/slide_hold_slide.py]: Here too.

File: examples/velocity_step_two_state.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import matplotlib.pyplot as plt
```
Comment (dopplershift) [inline examples/velocity_step_two_state.py]: Another one.

**Comment by dopplershift** [comment]
I think this is the right granularity. A shorter option for `plotting` would be `plots` or `plot`, but that's up to you.

**Comment by jrleeman** [comment]
Thinking about splitting up the tests by module as much as possible. Standard practice?

**Comment by dopplershift** [comment]
It's pretty standard to have one test file per module.

**Comment by jrleeman** [comment]
Ok. I've changed the naming a tiny bit and updated the examples/tests to reflect that. If it looks good to you, we'll merge and the test split-up can happen in another PR?

**Comment by dopplershift** [comment]
:+1:

**Pull request merged**